lib/urlwatch/command.py

```diff
@@ -143,15 +143,15 @@ def test_filter(self, id):
         return 0
 
     def prepare_jobs(self):
-        new_jobs = []
+        new_jobs = {*()}
```

Comment:
Funky. Just go with `set()` instead, I'd say. Same number of characters, and less confusing:

```suggestion
        new_jobs = set()
```